Macco-Academy/chirp
# Add App Icons

### What does this PR do?
This PR adds app icon to Chirp project

### Description of tasks to be completed
Add app icon to the project

### How to manually test
- checkout to branch
- run the app
- notice that the app icon is correct

### Any background context you want to provide
N/A

### Screenshots
<img src="https://github.com/Macco-Academy/chirp/assets/20377428/be1da3f5-c76c-4253-88f1-c1dedd4a9fd0" width=300 />
